### PR TITLE
Add the final translations for Portuguese

### PR DESCRIPTION
**Pull request by luispinho**
Add the final translations for Portuguese

### CODE REVIEW

File: apps/client/src/locales/messages.pt.xlf [hunk 1]
```diff
@@ -1259,55 +1259,55 @@
...
+        <target state="translated">Desempenho Bruto</target>
         <context-group purpose="location">
           <context context-type="sourcefile">apps/client/src/app/components/portfolio-summary/portfolio-summary.component.html</context>
           <context context-type="linenumber">63</context>
         </context-group>
       </trans-unit>
       <trans-unit id="e5c369abfbe1bd70f577aa03b2679797e38d7590" datatype="html">
         <source> Fees for <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transaction} other {transactions}}"/> </source>
-        <target state="new"> Fees for <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transaction} other {transactions}}"/> </target>
+        <target state="translated"> Taxas para <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transaction} other {transactions}}"/> </target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:1278]: Is `transaction` / `transactions` correct Portuguese?

File: apps/client/src/locales/messages.pt.xlf [hunk 2]
```diff
@@ -1259,55 +1259,55 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">apps/client/src/app/components/portfolio-summary/portfolio-summary.component.html</context>
           <context context-type="linenumber">78,81</context>
         </context-group>
       </trans-unit>
       <trans-unit id="272c7fd98af55bfa5b9d579176f1cfa25cd5489f" datatype="html">
         <source>{VAR_PLURAL, plural, =1 {transaction} other {transactions}}</source>
-        <target state="new">{VAR_PLURAL, plural, =1 {transaction} other {transactions}}</target>
+        <target state="translated">{VAR_PLURAL, plural, =1 {transaction} outras {transactions}}</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:1286]: Is `transaction` / `transactions` correct Portuguese?

**Review by dtslvr — CHANGES_REQUESTED** [review]
I have added two comments where I'm not sure if the translation is missing 🙂

File: apps/client/src/locales/messages.pt.xlf (hunk 1 above)
Comment (luispinho) [reply apps/client/src/locales/messages.pt.xlf]: Missed these ones, I've added them now

File: apps/client/src/locales/messages.pt.xlf (hunk 2 above)
Comment (luispinho) [reply apps/client/src/locales/messages.pt.xlf]: Missed these ones, I've added them now

**Review by dtslvr — APPROVED** [review]
Muito obrigado!

**Pull request merged**